### PR TITLE
fixed Refactor text color styling to be applied in the include file #1148

### PR DESCRIPTION
**Pull request by Sumitmaithani**
**Description**

This PR fixes [#1148 ](https://github.com/meshery/meshery.io/issues/1148)

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Sumitmaithani** [comment]
This PR can now be reviewed.

**Comment by vishalvivekm** [comment]
Great Work :clap:  @Sumitmaithani :rocket: 
Thanks for such a detailed description. 

@thisiskaransgit will you have a review and see that all your requested changes are incorporated. @Sumitmaithani has done a great work providing with the well detailed description of the fix

**Comment by Sumitmaithani** [comment]
@vishalvivekm Thanks for the kind words and appreciation, it means a lot! 🤗

**Review by thisiskaransgit — COMMENTED** [review]
1. Margin for the code component should be responsive 
2. in [metrics-store](https://meshery.io/catalog/wasm/metrics-store/) add a line break between the comments and commands

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
1. Margin for the code component should be responsive 
2. in [metrics-store](https://meshery.io/catalog/wasm/metrics-store/) add a line break between the comments and commands

**Comment by Sumitmaithani** [comment]
👋 Hey @thisiskaransgit , just wanted to give you an update that I've implemented the changes you suggested. 😊 Thanks for your valuable input!

https://github.com/meshery/meshery.io/assets/86047367/005598e0-c8f7-4aaa-885a-5bec72377594

**Review by GaganpreetKaurKalsi — DISMISSED** [review]
That's great work 💯 🚀 
Well done @Sumitmaithani  👏

**Comment by Sumitmaithani** [comment]
👋 Hye @leecalcote

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by Sumitmaithani** [comment]
@leecalcote 

shinchan

**Comment by thisiskaransgit** [comment]
Please take a look @Shivam-AfA  and @Chadha93, the changes look good to me

**Comment by vishalvivekm** [comment]
@Sumitmaithani 
Let's get the final feedback from the community: add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#), if you would :).
// @thisiskaransgit

**Comment by DarrenDsouza7273** [comment]
Amazing @Sumitmaithani 👍 Good work

**Review by Savio629 — CHANGES_REQUESTED** [review]
Can you make the 'Open in new tab' button more visible for dark mode like the close button?

File: _sass/single-page.scss [hunk 1]
```diff
@@ -3,6 +3,16 @@
   gap: 3rem;
 }
 
+.single-page code {
+  padding: 1rem;
+  border-radius: 0.5rem;
+  font-size: 0.9rem;
+  background-color: #1a2421;
```
Comment (Chadha93) [inline _sass/single-page.scss:10]: @thisiskaransgit Do we have a theme to reference for this website?

File: _sass/single-page.scss [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
+  line-height: 1rem;
+  color: #eee;
```
Comment (thisiskaransgit) [inline _sass/single-page.scss:12]:
```suggestion
  color: var(--color-primary-light);
```

File: _sass/single-page.scss (hunk 1 above)
Comment (thisiskaransgit) [inline _sass/single-page.scss:10]:
```suggestion
  background-color: var(--color-secondary-dark);
```

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: _sass/single-page.scss (hunk 1 above)
Comment (thisiskaransgit) [reply _sass/single-page.scss]: @Chadha93  PTAL

**Comment by Sumitmaithani** [comment]
> 

Thanks, @Savio629 ! Thanks for pointing out this issue, and sorry for the late response. I missed this message. I will fix it.

**Comment by Sumitmaithani** [comment]
Hey @Savio629, check it out now! 😊👀

https://github.com/meshery/meshery.io/assets/86047367/5f790704-763f-482b-b496-5b05f4b2c152

**Comment by Sumitmaithani** [comment]
Hey @Savio629, PTAL 😊👀

https://github.com/meshery/meshery.io/assets/86047367/5f790704-763f-482b-b496-5b05f4b2c152

**Review by Savio629 — APPROVED** [review]
LGTM 👍

**Comment by leecalcote** [comment]
@Avi-88 more duplication

**Comment by leecalcote** [comment]
@Sumitmaithani Thanks for working on this. 👍

**Pull request merged**